src/cmd-login.ts

```diff
@@ -11,14 +11,15 @@ import {
 import { parseEnv } from "./utils/env";
 import { encodeBasicAuth } from "./types/upm-config";
 import { Base64 } from "./types/base64";
-import { RegistryUrl, removeTrailingSlash } from "./types/registry-url";
+import { RegistryUrl } from "./types/registry-url";
 import {
   promptEmail,
   promptPassword,
   promptRegistryUrl,
   promptUsername,
 } from "./utils/prompts";
 import { CmdOptions } from "./types/options";
+import { removeTrailingSlash } from "./utils/string-utils";
```

Comment:
It seems the import is no longer needed.

Reply:
alright, I'll push a cleanup